pkg/ccl/backupccl/targets.go

```diff
@@ -443,84 +443,6 @@ func checkMissingIntroducedSpans(
 			}
 			tablesIntroduced[descpb.ID(tableID)] = struct{}{}
 		}
-
```

Comment:
i think we can get rid of this whole checkMissingIntroducedSpans function, right?